src/parsers/espresso.js

```diff
@@ -25,6 +30,468 @@ function toEspressoFormat(materialOrConfig) {
     );
 }
 
+/**
+ * @summary checks if the given fileContent is in the format of a Quantum ESPRESSO .in file.
+ * @param {String} fileContent
+ * @return {boolean}
+ */
+function isEspressoFormat(fileContent) {
+    const regex = /&CONTROL|&SYSTEM|CELL_PARAMETERS|ATOMIC_POSITIONS/i;
+    return regex.test(fileContent);
+}
+
+/**
+ * @summary Function to convert Fortran namelist to JavaScript object.
+ * @param {String} data - Fortran string.
+ * @return {Object}
+ */
+function parseFortranNameList(data) {
```

Comment:
This should go to `parsers.fortran`, perhaps?

Reply:
Maybe even to `code.js:utils` since this can be used elsewhere - by wode.js, for instance, later on